### PR TITLE
[Doc] Update doc for multiple api source

### PR DESCRIPTION
**Pull request by brynn-code**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .github/workflows/build_doc_ci.yml [hunk 1]
```diff
@@ -30,17 +29,15 @@ jobs:
...
-          setupType: ${{ env.packageSetupType }}
-          scriptPath: ${{ env.testWorkingDirectory }}
-
-      - name: Install tracing
+      - name: Install packages
         shell: pwsh
         run: |
           pip uninstall -y promptflow-tracing
-          pip install -e ${{ github.workspace }}/src/promptflow-tracing
+          pip install ${{ github.workspace }}/src/promptflow-tracing
```
Comment (brynn-code) [inline .github/workflows/build_doc_ci.yml:36]: Update publish doc ci after build doc ok

File: .github/labeler.yml [hunk 2]
```diff
@@ -3,21 +3,23 @@ examples:
...
+- src/promptflow-devkit/promptflow/_cli/**
 sdk:
-- src/promptflow/promptflow/_sdk/**
-- src/promptflow/promptflow/azure/**
+- src/promptflow-devkit/promptflow/_sdk/**
+- src/promptflow-azure/promptflow/azure/**
 promptflow:
-- src/promptflow/**
+- src/promptflow-core/**
+tracing:
```
Comment (wangchao1230) [inline .github/labeler.yml:12]: maybe use label named: promptflow-tracing

**Review by wangchao1230 — DISMISSED** [review]

File: .github/workflows/build_doc_ci.yml [hunk 3]
```diff
@@ -2,16 +2,15 @@ name: Build Doc CI
...
-#      - 'src/promptflow/promptflow/**'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'README.md'
+      - 'docs/**'
+      - 'scripts/docs/**'
+      - '.github/workflows/build_doc_ci.yml'
+      - 'src/promptflow/promptflow/**'
```
Comment (crazygao) [inline .github/workflows/build_doc_ci.yml:13]: trigger changes.

Comment (crazygao) [reply .github/workflows/build_doc_ci.yml:13]: also below

**Review by wangchao1230 — APPROVED** [review]

**Review by elliotzh — APPROVED** [review]

**Pull request merged**